### PR TITLE
pipecat 0.0.42 fixes

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

File: examples/foundational/07n-interruptible-google.py [hunk 1]
```diff
@@ -53,7 +53,6 @@ async def main():
         stt = DeepgramSTTService(api_key=os.getenv("DEEPGRAM_API_KEY"))
 
         tts = GoogleTTSService(
-            credentials=os.getenv("GOOGLE_CREDENTIALS"),
```
Comment (markbackman) [inline examples/foundational/07n-interruptible-google.py:56]: Don't you need either `credentials` or `credentials_path` to run this example? Should this be `credentials=os.getenv("GOOGLE_APPLICATION_CREDENTIALS")`?

File: src/pipecat/services/google.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
 except ModuleNotFoundError as e:
     logger.error(f"Exception: {e}")
     logger.error(
-        "In order to use Google AI, you need to `pip install pipecat-ai[google]`. Also, set `GOOGLE_API_KEY` environment variable."
+        "In order to use Google AI, you need to `pip install pipecat-ai[google]`. Also, set `GOOGLE_APPLICATION_CREDENTIALS` environment variable."
```
Comment (markbackman) [inline src/pipecat/services/google.py:43]: The `GOOGLE_API_KEY` is required to run the GoogleLLMService. So, I think this message needs to read: `Also, set `GOOGLE_API_KEY` for the `GoogleLLMService` and `GOOGLE_APPLICATION_CREDENTIALS` for the `GoogleTTSService`.

File: src/pipecat/services/cartesia.py [hunk 3]
```diff
@@ -344,7 +339,7 @@ async def run_tts(self, text: str) -> AsyncGenerator[Frame, None]:
                     voice_controls["emotion"] = self._settings["emotion"]
 
             output = await self._client.tts.sse(
-                model_id=self._model_id,
+                model_id=self._model_name,
```
Comment (markbackman) [inline src/pipecat/services/cartesia.py:342]: Thanks for catching this.

File: src/pipecat/services/gladia.py [hunk 4]
```diff
@@ -37,17 +37,17 @@
 def language_to_gladia_language(language: Language) -> str | None:
     match language:
         case Language.BG:
-            return "bg"
+            return "bulgarian"
```
Comment (markbackman) [inline src/pipecat/services/gladia.py:40]: Weird! For a websocket connection, they have very little documentation (that I can find). I found the language codes (ISO-639) for their REST endpoint, which listed the two letter codes. Maybe I'm just missing the docs. Thanks for catching this.

**Review by markbackman — APPROVED** [review]
LGTM.

Just a few minor comments about google.py and the corresponding foundational example. Otherwise, this looks good.

File: examples/foundational/07n-interruptible-google.py (hunk 1 above)
Comment (aconchillo) [reply examples/foundational/07n-interruptible-google.py:56]: No, by default Google uses `GOOGLE_APPLICATION_CREDENTIALS`

File: src/pipecat/services/google.py (hunk 2 above)
Comment (aconchillo) [reply src/pipecat/services/google.py:43]: Ah, OK.

**Pull request merged**